tests/Unit/API/Site/Controllers/MerchantCenter/ShippingTimeBatchControllerTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
```

Comment:
Maybe We can add some tests for:

- Countries with wrong format return Error
- Countries not supported can be added without errors.